seeds

```diff
@@ -0,0 +1 @@
+ba3bacc714817218562f743178228f23678b2873@5.83.160.108:26656,1e63e84945837b026f596ed8ae68708783d04ad4@51.75.145.123:26656,d2d452e7c9c43fa5ef017552688de60a5c0053ee@34.245.217.163:26656,dd36969b56c740bb40bb8badd4d4c6facc35dc24@206.189.115.41,a0aca8fb801c69653a290bd44872e8457f8b0982@47.99.180.54,27f8dd3bdbecbef7192291083706c156e523d8e0@3.122.248.21,aee0df1a660f301d456a0c2f805b372f7341e8ec@63.35.230.143:26656,7d1f660b361d6286715c098a3a171e554e9642bb@34.254.205.37,fa105c2291ac4aa452552fa4835266300a8209e1@88.198.41.62,bd410d4564f7e0dd9a0eb16a64c337a059e11b80@47.103.35.130
```

Comment:
Why do we have a seeds file?

Reply:
The seeds file was to avoid having a big old ugly list of seeds in readme.md, more or less.